docs/gl_objects/events.rst

```diff
@@ -2,6 +2,9 @@
 Events
 ######
 
+Events
+======
```

Comment:
isn't events now duplicated?

Reply:
Technically yes but without it the next heading isn't displayed with the proper size (see below)

We seem to also do this elsewhere, maybe for the same reason: https://github.com/python-gitlab/python-gitlab/blob/af781c10db3829163f977e494e4008acf2096d64/docs/gl_objects/projects.rst

![image](https://user-images.githubusercontent.com/16777978/115145897-56ddb700-a054-11eb-9bfa-3fa42e079c0f.png)